controls.py

```diff
@@ -144,7 +135,7 @@ def Sentence(*elems):
 
     return dbc.Form(groups)
 
-def Block(title, *body, width="100%"): #width of the block in its container
+def Block(title, *body, ison=True, width="100%"): #width of the block in its container
```

Comment:
What is `ison` short for here, and what is the use case of hiding the card in this way vs changing the style?

Reply:
ison stands for "is on." The [Onset Maproom](https://github.com/iridl/python-maprooms/blob/bd77571a6c27b15af4c13872f27f8f2ed26efa3a/enacts/onset/layout.py#L333) has an option to show cessation date analysis in addition to onset date analysis, but it's not available for all countries. So in countries where not available, the Block that provides the controls to define the parameters for cessation date is not displayed.

I think @aaron-kaplan wasn't too happy about that and would have preferred 2 Maprooms.

I didn't understand the last part of the question. I assume you mean hard-coding the style? well we can't in that set up since we use same layout.py and use config files to display it or not.

Reply:
Ok makes sense. Any way we could come up with a better name for this parameter? (even `is_on` would be fine) I don't want to be too picky I think it just might be confusing as is.